pkg/controllers/management/feature/feature_handler.go

```diff
@@ -11,8 +11,8 @@ import (
 	"k8s.io/apimachinery/pkg/runtime"
```

Comment:
Should this move to `pkg/api/controllers/` ? Thats where it's registered now, and I don't see anything in `pkg/controllers/management` using scaledContext.

Reply:
whoops, good catch